spec/models/registration_spec.rb

```diff
@@ -631,7 +631,7 @@
 
     it 'calls competition.attempt_auto_close! if reg is fully paid' do
       competition = create(:competition)
-      expect(competition).to receive(:attempt_auto_close!).exactly(1).times
+      expect(competition).to receive(:attempt_auto_close!)
```

Comment:
Just curious, why is the "exactly 1 time" assumption not working anymore? And can we reasonably compute what the new count should be instead?

Reply:
We can, and I've included it for both method calls! Not sure why one didn't have it before